### DISCUSSION TITLE
Running all tests cases in parallel, not group based on spec/feature file?

### DISCUSSION THREAD

**Discussion by wolfoo2931** [webdriverio/webdriverio#7923] [Q&A] [answered]
With max the maxInstances configuration I can configure how many spec/feature files should run in parallel. I'm using webdriver.io with jasmine.js and have multiple it statements (test cases) in one spec file (~30 spec files with 20 - 200 test cases). The test cases are quite unequally distributed across the spec files.

The question: Is there a configuration or plugin wich allows me to run tests in parallel, but not grouped in test files. All test cases - does not matter which spec file - can run at anytime.

Best Regards,
Oliver working on [MonsterWriter](https://www.monsterwriter.app/)

**Comment by christian-bromann** [comment, marked as answer]
@wolfoo2931 WebdriverIO can't run every single test within a file as a separate process. By design only every file is run that way. There have been some experiments with Cucumber to run every scenario in a different spec file.
